src/virt_deny.rs

```diff
@@ -1,76 +1,90 @@
+use std::sync::OnceLock;
+
 use anyhow::Result;
-use walrus::{Module, ValType};
+use walrus::{ExportItem, FuncParams, FuncResults, FunctionBuilder, Module, ValType};
 
 use crate::{
     virt_io::{stub_clocks_virt, stub_http_virt, stub_sockets_virt},
     walrus_ops::add_stub_exported_func,
 };
 
+/// Functions that represent the environment functionality provided by WASI clocks
+pub(crate) static WASI_CLOCK_FNS: OnceLock<Vec<(&str, FuncParams, FuncResults)>> = OnceLock::new();
+
+pub(crate) fn get_wasi_clock_fns() -> &'static Vec<(&'static str, FuncParams, FuncResults)> {
```

Comment:
This is kind of here-nor-there -- thought it might be nice to extract this stuff out, if possible.

Reply:
I like the refactoring, thank you. If you want to extend this out I'd be all for it!

Reply:
Will do!

- [ ] continue with refactoring of WASI fn arg lists